openedx/core/djangoapps/theming/helpers_dirs.py

```diff
@@ -0,0 +1,199 @@
+import os
```

Comment:
This file's intent is to contain all the directory-walking, theme-discovering code split into code that uses Django settings and code that does *not* use Django settings.